styles.css

```diff
@@ -0,0 +1,158 @@
+html {
+	font-family: "Trebuchet MS", sans-sarif;
+	box-sizing: border-box;
+	font-size: 10px;
+}
+
+body {
+	background-color:#0a415c;
+	margin: 0;
+	padding: 0;
+	color:#cfd2d4;
+}
+
+h2 {
+	font-size: 3rem;
+}
+
+h3 {
+	font-size: 2rem;
+}
+
+p {
+	font-size: 1.5rem;
+}
+
+nav {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-between;	
+	align-items: center;
+}
+
+.nav-bar {
+	width: 20%;
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	gap: 20px;
+	font-size: 2rem;
+	padding: 20px;
+}
+
+.nav-bar a:link, .nav-bar a:visited {
+	color: #cfd2d4;
+}
+.nav-bar a:hover {
+	color:#048230;
+}
+
+
+.cv-download-button {
+	width: 150px;
+	height: 30px;
+	border: 2px solid #048230;
+	background-color: #0a415c;
+	color: #048230;
+	font-size: 1.6rem;
+	cursor: pointer;
+	border-radius: 20px;
+	margin: 0 20px;
+}
+
+
+.cv-download-button:hover {
+	background-color:#048230;
+	color: #cfd2d4;
+}
+
+header {
+	position: relative;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	margin-bottom: 20px;
```

Comment:
it's better for this to be padding on the `main` element because visually it looks like it's part of the `main` element, so it would be easier to understand the code that way